apps/cloudinary2/src/locations/Field/Thumbnail.tsx

```diff
@@ -61,12 +69,51 @@ export function Thumbnail({ asset, isDisabled, onDelete }: Props) {
     transition,
   };
 
-  return (
-    <Card className={styles.card(isDisabled)} ref={setNodeRef} style={style} {...attributes}>
-      {url ? <img src={url} alt={alt} {...listeners} /> : <div {...listeners}>Asset not available</div>}
+  const consoleUrl = `https://console.cloudinary.com/console/media_library/query_search?q=${encodeURIComponent(
+    `{"userExpression":"(public_id = \\"${asset.public_id}\\")"}`,
+  )}`;
 
-      {!isDisabled && <IconButton variant="transparent" icon={<CloseIcon variant="muted" />} aria-label="Close" onClick={onDelete} className={styles.remove} />}
-    </Card>
+  return (
+    <div ref={setNodeRef}>
+      <AssetCard
+        style={style}
+        dragHandleRender={() => <DragHandle as="button" className={styles.dragHandle} label="Move card" {...attributes} {...listeners} />}
+        withDragHandle={!isDisabled}
+        src={url}
+        title={alt}
+        type="image"
+        icon={<img src={logo} alt="" width={24} height={24} />}
+        size="small"
+        actions={[
+          <MenuItem key="edit" as="a" href={consoleUrl} target="_blank" onClick={() => window.open(consoleUrl, "_blank")}>
+            Edit in Cloudinary <ExternalLinkIcon className={styles.menuItemIcon} />
+          </MenuItem>,
+          <MenuItem key="remove" onClick={onDelete} isDisabled={isDisabled}>
+            Remove
+          </MenuItem>,
+          <MenuDivider key="divider" />,
+          <Menu.SectionTitle key="file-information-title">File information</Menu.SectionTitle>,
+          <MenuItem key="file-information" className={styles.fileInformation.menuItem} isDisabled>
+            <dl className={styles.fileInformation.dl}>
+              <dt>Location:</dt>
+              <dd>{asset.public_id.split('/').slice(0, -1).join('/') || 'Home'}</dd>
+              <dt>Format:</dt>
+              <dd>{asset.format}</dd>
+              <dt>Size:</dt>
+              <dd>{fileSize(asset.bytes).human('jedec')}</dd>
+              <dt>Dimensions:</dt>
+              <dd>
+                {asset.width} x {asset.height} px
```

Comment:
Small nit: designs do not have a unit specified for dimensions, maybe remove the `px` ?

Reply:
Seems like the designs are updated 🤔  Josh asked me to add the unit.